lib/units/ios-device/plugins/wda/WdaClient.js

```diff
@@ -785,13 +785,12 @@ module.exports = syrup.serial()
         ))
       ])
 
-      //TODO: handle negative case when device type not detected. Seems lifecycle.fatal(err) is ok.
       WdaClient.getDeviceType()
```

Comment:
either device type so size/scale should be skipped if already present in DB
@ignacionar ^^^